src/inc/corjit.h

```diff
@@ -113,18 +113,22 @@ class CORJIT_FLAGS
         CORJIT_FLAG_USE_AVX                 = 14,
         CORJIT_FLAG_USE_AVX2                = 15,
         CORJIT_FLAG_USE_AVX_512             = 16,
-        CORJIT_FLAG_FEATURE_SIMD            = 17,
 
     #else // !defined(_TARGET_X86_) && !defined(_TARGET_AMD64_)
 
         CORJIT_FLAG_UNUSED6                 = 13,
         CORJIT_FLAG_UNUSED7                 = 14,
         CORJIT_FLAG_UNUSED8                 = 15,
         CORJIT_FLAG_UNUSED9                 = 16,
-        CORJIT_FLAG_UNUSED10                = 17,
 
     #endif // !defined(_TARGET_X86_) && !defined(_TARGET_AMD64_)
 
+    #if defined(_TARGET_XARCH_) || defined(_TARGET_ARM64_)
```

Comment:
I don't think you can use `_TARGET_XARCH_` in corjit.h because it is included places that don't define that. So, to be consistent, it might make sense not to use it in jitee.h either.

Reply:
@BruceForstall done